lib/hydra/works/models/concerns/generic_file/characterization.rb

```diff
@@ -0,0 +1,116 @@
+module Hydra::Works::GenericFile
+  module Characterization
+    extend ActiveSupport::Concern
+
+    autoload :FitsDatastream, 'hydra/works/models/generic_file/characterization/fits_datastream.rb'
```

Comment:
@mjgiarlo Hydra-Works depends on ActiveFedora which depends on ActiveSupport.  Yes, that models makes it tricky. I think we have a structural problem. A module in `concerns/` probably should not define other modules that are not under `concerns/`.  For the rest of them (Image, Document, Video, Audio), it should be autoloading relative to the current file, so you shouldn't have to worry about it.